backend/src/test/java/capstone/backend/controller/OrderToCustomerControllerTest.java

```diff
@@ -0,0 +1,71 @@
+package capstone.backend.controller;
+
+import capstone.backend.CombinedTestContainer;
+import capstone.backend.repo.*;
+import capstone.backend.utils.ControllerTestUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static org.junit.jupiter.api.Assertions.*;
+@Testcontainers
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class OrderToCustomerControllerTest {
+
+    @Autowired
+    OrderToCustomerController controller;
```

Comment:
why not set to private?

Reply:
good question. I don't know. It appears in all my controller test I have not set the imported components to private. Have we usually done this? I don't remember.